image/manifest_test.go

```diff
@@ -122,3 +103,45 @@ func TestUnpackLayer(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func createTarFile(name string, list []tarContent) (descriptor, error) {
```

Comment:
We use tar files for image-layout as well.  This helper is about creating layers, so it should probably be `createLayer` or similar to distinguish from image-layout tooling.